pkg/fuzzy/fuzzy.go

```diff
@@ -5,6 +5,8 @@ import (
 	"fmt"
 	"strings"
 
+	"golang.org/x/text/language"
+
```

Comment:
Please remove the empty line. Where is the empty line coming from? Do you add imports manually?